.pre-commit-config.yaml.jinja

```diff
@@ -122,6 +122,13 @@ repos:
     rev: {{ repo_rev.setuptools_odoo }}
     hooks:
       - id: setuptools-odoo-make-default
+      - id: setuptools-odoo-get-requirements
```

Comment:
I think this should be wrapped by an `{% if dependency_installation_mode == 'PIP' %}` block, right?

Reply:
@Yajo I don't think so: this just ensures that `requirements.txt` is consistent with the declarations in manifests (and possible overrides in setup.py), which must be the case in both installation modes. 

Generating `oca_dependencies.txt` (#32) will be for `MQT_DEP=PIP` only, of course.

Reply:
Although generating requirements.txt is useful regardless of the installation mode, we don't want to activate it by default on 13.0 branches since it could make repos with imperfect manifests red. So I added a question to control this. @Yajo